CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+Contributing to the learning material
+=====================================
+
+Writing Guidelines
+------------------
+
+- Text must contain up to 79 characters per line.
+
+- Source-code examples should be complete, compilable, and free of errors.
+
+    - Valid exceptions for incomplete code are, for example, when the
```

Comment:
I actually disagree with this style, and it is not used in the intro-to-ada course to the best of my knowledge

Reply:
It _is_ actually used in the course in some places. Just look for code blocks marked with `ada-nocheck`: some of them are referring to previous examples, highlighting one statement from that example and then saying something like "we could rewrite that statement in the previous example like this: [_incomplete code here containing single line_]". In my opinion, this is a valid exception to the rule; if we repeat the same code from the previous example once more just to change one single line, then it could be more confusing to the reader.